keg_elements/db/utils.py

```diff
@@ -83,13 +82,15 @@ def _validate_unique_msg(dialect, msg, constraint_name=None):
 
 @contextlib.contextmanager
 def raises_unique_exc(constraint_name):
+    import pytest
     with pytest.raises(sa.exc.IntegrityError) as exc:
         yield
     assert validate_unique_exc(exc.value, constraint_name)
 
 
 @contextlib.contextmanager
 def raises_check_exc(contraint_name):
+    import pytest
     with pytest.raises(sa.exc.IntegrityError) as exc:
         yield
     assert _is_check_const(db.engine.dialect.name, str(exc.value), contraint_name)
```

Comment:
Instead of tucking imports into the code, would it make more sense to just extract these two methods out to `testing.py`?